modules/coreboot

```diff
@@ -29,16 +29,17 @@ else ifeq "$(CONFIG_COREBOOT_VERSION)" "4.13"
 	coreboot_hash := 4779da645a25ddebc78f1bd2bd0b740fb1e6479572648d4650042a2b9502856a
 	coreboot-blobs_hash := 060656b46a7859d038ddeec3f7e086e85f146a50b280c4babec23c1188264dc8
 	coreboot_depends := $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
-else ifeq "$(CONFIG_COREBOOT_VERSION)" "4.15"
-	coreboot_version := 4.15
-	coreboot_hash := 20e6aaa6dd0eaec7753441c799711d1b4630e3ca709536386f2242ac2c8a1ec5
-	coreboot-blobs_hash := c0e2d8006da226208ba274a44895d102cb2879cf139cc67bba5f62e67b871f6d
+else ifeq "$(CONFIG_COREBOOT_VERSION)" "4.17"
+	coreboot_version := 4.17
+	coreboot_hash := 95da11d1c6a450385101a68799258a398ce965f4e46cce6fe8d5ebd74e50c125
+	coreboot-blobs_hash := a2277fe7a2b2aab5da0aa335158460e00b852382f6736f2179992805702eb607
 	coreboot_depends := $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
 	EXTRA_FLAGS := -fdebug-prefix-map=$(pwd)=heads -gno-record-gcc-switches -Wno-error=packed-not-aligned -Wno-error=address-of-packed-member
 else ifeq "$(CONFIG_COREBOOT_VERSION)" "talos_2"
-	coreboot_version = git
-	coreboot_commit_hash = 2207bbcccba31ad89cf21607b0d8d05d8dc47c03
-	coreboot_repo := https://github.com/Dasharo/coreboot
+    coreboot_version = git
+    coreboot_commit_hash = 2207bbcccba31ad89cf21607b0d8d05d8dc47c03
+    coreboot_repo := https://github.com/Dasharo/coreboot
+
```

Comment:
@danielp96 Your editor might have tweaked the indentation on these lines

Reply:
Didn't notice, gonna fix.